WordPress/src/main/res/layout/post_list_item_compact.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
You might want to consider checking the dimens.xml for existing values and reusing them or creating new values so we don't have hardcoded dimensions in layout files. Wdyt?

Reply:
Sure, I'll resolve this. Good point, and my bad.

Reply:
👍